From 66e8253cd495153e6a6f51a45625348cd7f69994 Mon Sep 17 00:00:00 2001
From: Segey Lapin <slapin@ossfans.org>
Date: Sat, 27 Jul 2019 16:59:35 +0300
Subject: [PATCH] LLVM fix for new gcc

---
 include/llvm/IR/ValueMap.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/llvm/IR/ValueMap.h b/include/llvm/IR/ValueMap.h
index 4d00b63..d7c8b0c 100644
--- a/include/llvm/IR/ValueMap.h
+++ b/include/llvm/IR/ValueMap.h
@@ -99,7 +99,7 @@ public:
   explicit ValueMap(const ExtraData &Data, unsigned NumInitBuckets = 64)
       : Map(NumInitBuckets), Data(Data) {}
 
-  bool hasMD() const { return MDMap; }
+  bool hasMD() const { return static_cast<bool>(MDMap); }
   MDMapT &MD() {
     if (!MDMap)
       MDMap.reset(new MDMapT);
-- 
2.16.1

